docs/roles/RoleNORManager.md

```diff
@@ -6,17 +6,24 @@ title: New Ontology Request (NOR) Manager
 
 The New Ontology Request (NOR) Manager has these basic duties:
 
-1. Receiving `new ontology` requests on OBO and acknowledging the receipt
-2. Including the ontology in the OBO New Ontology Request (NOR) Dashboard
+1. Receiving `new ontology` requests on the OBO Foundry GitHub issues tracker and acknowledging the receipt
+2. Including the ontology in the OBO New Ontology Request (NOR) Dashboard and updating the dashboard
 3. Supporting the New Ontology requestor to pass the OBO Dashboard
 4. Assigning a provisional reviewer from the OBO Operations Committee
 5. Removing the accepted ontology from the NOR Dashboard upon acceptance
 
 
 ### Receiving `new ontology` requests on OBO and acknowledging the receipt
 
-- The NOR Manager MUST actively listen to issues on the [OBO Issue Tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
-- When a new ontology request is received, and the wrong issue template was used, the New Ontology Request (NOR) Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue.
+- The NOR Manager MUST actively listen to issues on the [OBOFoundry GitHub issues tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
+- When a new ontology request is received, and the wrong issue template was used, the NOR Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue with the following message:
+    ```markdown
+    Dear @GITHUBNAME,
+    
+    Thank you for your submission. 
+    However, the wrong issue template has been used. Please make sure to follow the [New Ontology Submission Instructions](https://obofoundry.org/docs/NewOntologyRegistrationInstructions.html) and use the [New Ontology Request issue template](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?assignees=&labels=new+ontology&template=new-ontology-request.md&title=).
```

Comment:
This link goes to a blank issue, not a template (see screenshot). Is that expected?
![image](https://user-images.githubusercontent.com/47164199/210031118-8b8b56b9-3c82-4c1f-a46d-6a49086cd47e.png)

Reply:
We should use this lin:

```
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/new?assignees=&labels=new+ontology&template=new-ontology.yml&title=Request+for+new+ontology+%5BNAME%5D
```

Reply:
Document updated with the new link